src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -748,6 +792,16 @@ public virtual void OnNodeModified(bool forceExecute = false)
             if (handler != null) handler(this);
         }
 
+        public event Action<NodeModel> ToggleNodeFreeze;
+        public virtual void OnToggleNodeFreeze()
+        {
+            if (!RaisesModificationEvents)
```

Comment:
This is sort of a misuse of this boolean. It was only meant to silence `NodeModified` events. If you're generalizing it, you should update the documentation of this property appropriately.